core/runtime/src/main/java/io/quarkus/runtime/logging/CategoryConfig.java

```diff
@@ -1,20 +1,32 @@
 package io.quarkus.runtime.logging;
 
-import io.quarkus.runtime.annotations.ConfigGroup;
-import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.*;
```

Comment:
Can you please convert this to non-start imports?

Reply:
Done